docs/conf.py

```diff
@@ -0,0 +1,28 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# For the full list of built-in configuration values, see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Project information -----------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
+
+project = 'Fromager'
+copyright = '2024, Doug Hellmann'
+author = 'Doug Hellmann'
+release = '0.21.3'
```

Comment:
Let's use https://docs.python.org/3/library/importlib.metadata.html to get the version from the package metadata.

Reply:
I tried using it but Sphinx is detecting it as function for some reason and I get the error:
`TypeError: expected string or bytes-like object, got 'function'`

Code:
```
from importlib.metadata import version
project = "Fromager"
copyright = "2024, Doug Hellmann"
author = "Doug Hellmann"
release = version("fromager")

```

Reply:
I bet something is looking at `version` attribute of the module

do `from importlib import metadata`